src/main/scala/io/moia/protos/teleproto/PbResult.scala

```diff
@@ -99,4 +105,13 @@ object PbFailure {
 
   def apply(message: String): PbFailure =
     apply("", message)
+
+  /**
+    * Collects and combines all the errors of all failures.
+    */
+  def combine(results: PbResult[_]*): PbFailure =
```

Comment:
Not 100% sure: if all results would be PbSuccess the result should be PbSuccess or?

Reply:
I added a comment that explains the usage. It cannot be a `PbSuccess` without a combining method and that's not the intention of that helper.